app/forms/questionnaire_form.py

```diff
@@ -57,6 +57,14 @@ def __init__(
 
         super().__init__(**kwargs)
 
+    @property
+    def data(self):
+        d = super().data
+        if d["csrf_token"] is None:
+            d["csrf_token"] = ""
+
+        return d
```

Comment:
`csrf_token` is now parsed as `None` if non existent, I'm not sure of the impact so I convert it back to blank string here to be safe but it is dirty...